pkg/reconciler/trigger/trigger_test.go

```diff
@@ -157,8 +157,9 @@ func init() {
 
 func TestReconcile(t *testing.T) {
 	brokerConfigs := map[string]*duckv1.KReference{
-		"rabbitmqClusterConfig": configWithRabbitMQCluster(),
-		"rabbitmqBrokerConfig":  configWithRabbitMQBrokerConfig(),
+		"rabbitmqClusterConfig":     configWithRabbitMQCluster(),
+		"rabbitmqClusterConfigNoNS": configWithRabbitMQBrokerConfigNoNS(),
```

Comment:
Same as above. This isn't a large enough change to warrant the entire table of tests to be run where the namespace is missing.